Source/Utils/WarpXAlgorithmSelection.cpp

```diff
@@ -93,6 +93,16 @@ const std::map<std::string, int> ParticleBCType_algo_to_int = {
     {"default",    ParticleBoundaryType::Absorbing}
 };
 
+const std::map<std::string, int> ExternalFieldType_algo_to_int = {
+    {"all", ExternalFieldType::All},
```

Comment:
Can we change this to "allexternal" and "ExternalFIeldType::AllExternal"
using just 'all' is a little too generic and will be hard to search for